com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/Varyings.hlsl

```diff
@@ -105,7 +105,8 @@ Varyings BuildVaryings(Attributes input)
 #endif
 
 #ifdef VARYINGS_NEED_VIEWDIRECTION_WS
-    output.viewDirectionWS = GetWorldSpaceNormalizeViewDir(positionWS);
+    // Need the unnormalized direction here as it will get normalized after interpolation
```

Comment:
Would be a stronger warning as:   Need the unnormalized direction here as otherwise interpolation is incorrect.  It is normalized after interpolation in the fragment shader.

Reply:
I basically had that originally, but thought it was a long comment for a single line of code.  I'll put it back :-)